src/hooks/useVenueScheduleEvents.ts

```diff
@@ -114,7 +110,8 @@ const useVenueScheduleEvents = ({
     fromUnixTime(firstRangeDateInSeconds)
   );
 
-  const dayDifference = getEventDayRange(daysInBetween, isOneEventAndLive);
+  // add 1 day to daysInBetween to form a full timeline
```

Comment:
This is just English for the line under, adds no more meaning. Would be nice to instead have the reason of why there is a need to add a day.

Reply:
changed the comment: https://github.com/sparkletown/sparkle/pull/2720/commits/658dc03ac29ab71d1ea74795e3bd5e23be09a05e